main.go

```diff
@@ -91,8 +111,20 @@ func main() {
 	for _, entry := range entries[:numFilled] {
 		logger := log.WithField("repo", entry)
 
+		matched, err := regexp.MatchString(*repoRegexp, entry)
+
+		if matched == false {
+			logger.WithFields(log.Fields{"entry": entry}).Debug("Ignore non matching repository (-repo=", *repoRegexp, ")")
+			continue
+		}
+
 		// Establish repository object in registry
-		repoName, _ := reference.ParseNamed(entry)
+		repoName, err := reference.WithName(entry)
```

Comment:
I didn't check the API again, but do you exactly know what the difference is between `ParsedName` and `WithName`?